### DISCUSSION TITLE
How to specify port for command

### DISCUSSION THREAD

**Discussion by ericzlot** [Waboodoo/HTTP-Shortcuts#308] [Q&A]
I'm trying to enter a command for my Sony TV and I'm having trouble figuring out how to configure the shortcut. Below is the command with the ip and port

http://192.168.0.5:20060/*SCPOWR0000000000000000

Any help on how to get a shortcut made to send this command?

**Comment by Waboodoo** [comment]
This looks like you can just create a regular HTTP Shortcut for it. The URL you have there already contains the port. You can create a shortcut for it via the following steps:

1. Open the app
2. Tap the + button at the bottom right
3. Select "Regular HTTP Shortcut"
4. GIve it a name, and optionally also a description and change the icon
5. Tap "Basic Request Settings"
6. On the screen that opens, enter your full URL into the URL field, so in your case http://192.168.0.5:20060/*SCPOWR0000000000000000
7. I don't know what method your TV expects, but it is likely GET or POST. I suggest you leave it on GET and if that doesn't work try with POST or PUT.
8. Go back and test your shortcut by tapping the play button at the top right.
9. Optionally, you can also open the "Response Handling" screen here and change how the response should be displayed. Most likely you'll want to change the "Display Type" to "Toast Popup" and the "On Success" dropdown to "Show a message".
10. Save your changes via the checkmark button at the top right.
11. You now have a shortcut that hopefully does what it should do.
